Tests/Twig/Extension/UploaderExtensionTest.php

```diff
@@ -27,7 +27,10 @@ public function testGetName()
 
     public function testAssetIsRegistered()
     {
-        $this->assertArrayHasKey('vich_uploader_asset', $this->extension->getFunctions());
+        $functions = $this->extension->getFunctions();
+        $expectedFunction = new \Twig_SimpleFunction('vich_uploader_asset', array($this->extension, 'asset'));
+
+        $this->assertEquals(array($expectedFunction), $functions);
```

Comment:
This equality test is meaningless IMHO.

We should test that the extension defines exactly one function and that this function is named `vich_uploader_asset`.
This would make the test-case look like this:

```php
$this->assertCount(1, $functions);
$this->assertEquals('vich_uploader_asset', $functions[0]->getName());
```